src/account/Settings.tsx

```diff
@@ -411,143 +365,139 @@ export class Account extends React.Component<Props, State> {
     }
   }
 
-  render() {
-    const { t, i18n, numberVerified } = this.props
-    const promptConfirmRemovalModal = this.props.route.params?.promptConfirmRemovalModal ?? false
-    const currentLanguage = locales[i18n.language]
-
-    return (
-      <SafeAreaView style={styles.container}>
-        <DrawerTopBar />
-        <ScrollView testID="SettingsScrollView">
-          <TouchableWithoutFeedback onPress={this.onDevSettingsTriggerPress}>
-            <Text style={styles.title} testID={'SettingsTitle'}>
-              {t('settings')}
-            </Text>
-          </TouchableWithoutFeedback>
-          <View style={styles.containerList}>
-            <SettingsItemTextValue
-              testID="EditProfile"
-              title={t('editProfile')}
-              onPress={this.goToProfile}
-            />
-            {!numberVerified && (
-              <SettingsItemTextValue title={t('confirmNumber')} onPress={this.goToConfirmNumber} />
-            )}
-            <SettingsItemTextValue
-              title={t('languageSettings')}
-              testID="ChangeLanguage"
-              value={currentLanguage?.name ?? t('unknown')}
-              onPress={this.goToLanguageSetting}
-            />
+  const currentLanguage = useSelector(currentLanguageSelector)
```

Comment:
nit: Move this to the top of the function where all the selectors are.

Reply:
ah, thank you!! i forgot about this